StandardConfig/production/ILDReconstruction.py

```diff
@@ -85,35 +91,33 @@
     default="ILD_l5_o1_v02",
 )
 parser.add_argument(
-    "--perfectPFA", help="Run perfect PandoraPFA", action="store_true", default=False
+    "--perfectPFA",
+    help="Run perfect PandoraPFA",
+    action="store_true",
 )
 parser.add_argument(
     "--runOverlay",
     help="Run background overaly. NOTE: You have to make sure that the Overlay algorithms in "
     " BgOverlay/BgOverlay.py are provided with the necessary overlay files",
     action="store_true",
-    default=False,
-)
-# BeamCal reco configuration
-parser.add_argument(
-    "--runBeamCalReco",
-    help="Run the BeamCal reco",
-    action="store_true",
-    dest="runBeamCalReco",
```

Comment:
Please keep this argument. This is done this way because the original Marlin steering file allowed to explicitly set this.